### DISCUSSION TITLE
建议对密码设置增加一些提示/警告

### DISCUSSION THREAD

**Discussion by UGYnpU2nuB8QCFqDrsWZgmxMBp43WQ** [bohanwood/debi#85] [Ideas]
某些特殊字符无法作为脚本所设置的密码，若有人误设置会导致SSH密码无法校验。

**Comment by gubindp** [comment]
老哥。请教个问题。我在使用这个dd 时候设置了简单数字密码。请问有没有什么方法获得原始密码？或者有没有其他办法连接
